client/docs/static/openapi/openapi.yaml

```diff
@@ -28473,7 +28473,7 @@ paths:
         required: true
       responses:
         "202":
-          description: Tx was succesfully generated
+          description: Tx was successfully generated
```

Comment:
I think that this file is auto-generated. We should find the source and fix it there